pkg/didcomm/packer/authcrypt/pack.go

```diff
@@ -33,7 +33,7 @@ import (
 const (
 	encodingType = "didcomm-envelope-enc"
 	// ThirdPartyKeysDB is a store name containing keys of third party agents.
-	ThirdPartyKeysDB = "thirdPartyKeysDB"
+	ThirdPartyKeysDB = "thirdpartykeysdb"
```

Comment:
what's the reason for this change?

Reply:
couchdb will not accept upper case

Reply:
You are right: https://docs.couchdb.org/en/stable/api/database/common.html#put--db.

Patching these sorts of db-specific bugs everywhere is untenable.

Reply:
weird indeed, but I'm ok with all lower case name for this store used specifically for the new Authcrypt packer.